src/util/log.d

```diff
@@ -345,9 +323,12 @@ class FileLogger(Layout) : Logger
         file.flush;
     }
 
-    override Nullable!int fileDescriptor() const @nogc nothrow @safe
+    // Segfault handling requires limiting ourselves to signal-safe functions,
+    // which is a very limited subset - Phobos/toString/anything GC is right out.
+    // For such cases, offer backdoor access to the file handle.
+    public int __fileDescriptor() const @nogc nothrow @safe
```

Comment:
I would just omit the leading underscores.
Why should a file logger not make its file descriptor readable?

Reply:
No prob. I wanted to avoid these APIs being casually used. But I guess it's not an issue.